### ISSUE TITLE
The training question about diffste.

### ISSUE THREAD

**Issue by thebestYezhang** [UCSB-NLP-Chang/DiffSTE#12]
Thanks for your great work! I have a question. When I train the diffste in my own dataset, the training Progress bar only prints out the loss of half of total training steps, and the loss of the left half of total training steps is not show by the Progress bar. I carefully reviewed the training process, but couldn't find the problem. May I ask if you were in the same situation at training time?
![image](https://github.com/UCSB-NLP-Chang/DiffSTE/assets/49755801/d7526ea7-e8bf-4d68-a113-e8a7a7b576b0)

**Comment by Question406** [comment]
Hi, actually, the training loss printed in the terminal is the loss for a single batch instead of the average loss over the entire dataset. For the details of what is printed, please see [this link](https://lightning.ai/docs/pytorch/stable/common/progress_bar.html#tqdmprogressbar).

The reason this progress bar is split in the middle of one epoch is the output during training, the line `do_classifier_free_guidance`; I guess this is some log information you print when you log some generated images.

**Comment by yang-chenyu104** [comment]
i want to say format of data,i want to train text editing

**Comment by yang-chenyu104** [comment]
when i training model,the path of log makes me very sad

**Comment by HHanTheKing** [comment]
> Thanks for your great work! I have a question. When I train the diffste in my own dataset, the training Progress bar only prints out the loss of half of total training steps, and the loss of the rest half of total training steps is not show by the Progress bar. I carefully reviewed the training process, but couldn't find the problem. May I ask if you were in the same situation at training time? ![image](https://github.com/UCSB-NLP-Chang/DiffSTE/assets/49755801/d7526ea7-e8bf-4d68-a113-e8a7a7b576b0)

Hello, I also want to use my own dataset for fine-tuning. What is the structure of your dataset (the training data provided by the blogger is quite complex). Is it in the form of a triple of mask-original image-image to be generated? How many images are there in each group?

**Comment by HHanTheKing** [comment]
> Thanks for your great work! I have a question. When I train the diffste in my own dataset, the training Progress bar only prints out the loss of half of total training steps, and the loss of the rest half of total training steps is not show by the Progress bar. I carefully reviewed the training process, but couldn't find the problem. May I ask if you were in the same situation at training time? ![image](https://github.com/UCSB-NLP-Chang/DiffSTE/assets/49755801/d7526ea7-e8bf-4d68-a113-e8a7a7b576b0)

Hello, I also want to use my own dataset for fine-tuning. What is the structure of your dataset (the training data provided by the blogger is quite complex). Is it in the form of a triple of mask-original image-image to be generated? How many images are there in each group? I hope to get your reply, thank you!
